eng/pipelines/runtime.yml

```diff
@@ -109,7 +109,9 @@ jobs:
       - eng/pipelines/installer/*
     - subset: runtimetests
       include:
+      - src/tests/*
       - src/coreclr/tests/*
+      - src/coreclr/build-test.cmd
```

Comment:
aren't we going to move this scripts to the new folder?

Reply:
Ultimately, yes, but that would again require additional changes to the pipelines. As mentioned above, I would prefer to tackle this as a second step once the bulk test move has been merged in.

Reply:
Makes sense.